demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -479,22 +918,72 @@
    "outputs": [],
    "source": [
     "# without background\n",
-    "cnm.estimates.play_movie(images, q_max=99.9, magnification=2,\n",
-    "                                 include_bck=False, gain_res=4, bpx=bord_px);"
+    "cnmfe_model.estimates.play_movie(images, \n",
+    "                                 q_max=99.9, \n",
+    "                                 magnification=2,\n",
+    "                                 include_bck=False,\n",
+    "                                 gain_res=5,\n",
+    "                                 use_color=True);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A couple of things to note about that movie:\n",
+    "- The middle panel of neural activity includes *all* components (accepted and rejected), so you will see some of the blood vessel \"activity\" that was discovered and later rejected.\n",
+    "- The residual includes some activity that looks neural in origin. You can try playing with different params to get them (which would you try?). The [Mesmerize](https://github.com/nel-lab/mesmerize-core) package is a great way to search parameter space and visualize results with more sophisticated visualization tools if you have a tricky data set."
```

Comment:
"which would you try?" -- unclear what this is meant to add

Reply:
Creativity juicer.